src/components/Avatar.jsx

```diff
@@ -2,13 +2,18 @@ import React from 'react'
 
 import UiAvatar from 'cozy-ui/transpiled/react/Avatar'
 
-import { pickPurposeIcon, purposeToColor } from 'src/components/helpers'
+import {
+  pickPurposeIcon,
+  purposeToColor,
+  pickModeIcon,
+  modeToColor
+} from 'src/components/helpers'
 import { OTHER_PURPOSE } from 'src/constants/const'
 
 const makeStyle = ({ faded, color }) => {
   return faded
     ? {
-        color: 'var(--primaryColor)',
+        color: color ? color : 'var(--primaryColor)',
```

Comment:
```suggestion
        color: color || 'var(--primaryColor)',
```

Reply:
nit: does this feature deserve a test?

Reply:
le makeStyle en lui-même est super light donc je pense que non.

Ca soulève la question de tests visuels, à réfléchir.